ui/components/component-library/avatar-account/avatar-account.js

```diff
@@ -4,25 +4,36 @@ import PropTypes from 'prop-types';
 import Jazzicon from '../../ui/jazzicon/jazzicon.component';
 import BlockieIdenticon from '../../ui/identicon/blockieIdenticon/blockieIdenticon.component';
 import { AvatarBase } from '../avatar-base';
-
 import { SIZES } from '../../../helpers/constants/design-system';
-import { DIAMETERS, TYPES } from './avatar-account.constants';
 
-export const AvatarAccount = ({ size, address, className, type, ...props }) => {
+import {
+  DIAMETERS,
+  TYPES,
+  AVATAR_ACCOUNT_SIZES,
+} from './avatar-account.constants';
+
+export const AvatarAccount = ({
+  size = SIZES.MD,
+  address,
+  className,
+  type,
+  ...props
+}) => {
   return (
     <AvatarBase
       size={size}
-      className={classnames('avatar-account', className)}
+      className={classnames('mm-avatar-account', className)}
       {...props}
     >
       {type === 'Jazzicon' ? (
         <Jazzicon
-          className={classnames('avatar-account__jazzicon')}
+          className={classnames('mm-avatar-account__jazzicon')}
           address={address}
-          diameter={DIAMETERS[size]}
+          diameter={Number(DIAMETERS[size])}
```

Comment:
Does this fix this console issue?

![Screenshot 2022-12-13 at 3 05 42 PM](https://user-images.githubusercontent.com/8112138/207464621-3ea181dc-527a-468b-8bf0-639e22a0c4f8.png)

Reply:
Yup